v2/emailpassword/common-customizations/sessions/token-transfer-method.mdx

```diff
@@ -0,0 +1,314 @@
+---
+id: token-transfer-method
+title: Token transfer methods
+hide_title: true
+show_ui_switcher: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./session/common-customizations/sessions/token-transfer-method.mdx -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import TabItem from '@theme/TabItem';
+import NpmOrScriptTabs from "/src/components/tabs/NpmOrScriptTabs"
+import AngularUIImplementation from "/src/components/reusableSnippets/angularUIImplementation"
+import {Answer}from "/src/components/question"
+import {PreBuiltOrCustomUISwitcher, PreBuiltUIContent, CustomUIContent} from "/src/components/preBuiltOrCustomUISwitcher"
+import FrontendPreBuiltUITabs from "/src/components/tabs/FrontendPreBuiltUITabs"
+import FrontendCustomUITabs from "/src/components/tabs/FrontendCustomUITabs"
+import FrontendMobileSubTabs from "/src/components/tabs/FrontendMobileSubTabs"
+
+<!-- TODO: re-think title -->
+# Token transfer methods
+
+:::caution
+We recommend cookie-based sessions in browsers because header-based sessions require saving the access and refresh tokens in storage vulnerable to XSS attacks.
+:::
+
+When creating/authorising sessions, the SDK has to choose to send the tokens to the frontend by cookies or custom headers. This choice is ultimately controlled by the backend, but it follows a preference set in the frontend configuration.
+
+## Frontend configuration
+
+You can provide a `tokenTransferMethod` property in the configuration of the Session recipe to set the preferred token transfer method, which will be sent to the backend with every request in the `st-auth-mode` header.
+
+By default, the backend follows this preference.
+
+We recommend using cookie-based sessions in browsers and header-based sessions in mobile apps and environments where you can't or prefer not to use the built-in cookie manager.
+
+<PreBuiltOrCustomUISwitcher>
+
+<PreBuiltUIContent>
+
+<FrontendPreBuiltUITabs>
+
+<TabItem value="reactjs">
+
+```tsx
+import SuperTokens from "supertokens-auth-react";
+import Session from "supertokens-auth-react/recipe/session";
+
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    recipeList: [
+        Session.init({
+            // highlight-next-line
+            tokenTransferMethod: "header"
```

Comment:
you should specify all possible values for this - cause some people use JS and they won't get type hint